code/modules/projectiles/ammo_types/energy_ammo.dm

```diff
@@ -74,6 +74,51 @@
 			return
 		X.use_plasma(0.3 * X.xeno_caste.plasma_max * X.xeno_caste.plasma_regen_limit) //Drains 30% of max plasma on hit
 
+#define BFG_SOUND_DELAY_SECONDS 1
+/datum/ammo/energy/bfg
+	name = "bfg glob"
+	icon_state = "bfg_ball"
+	hud_state = "electrothermal"
+	hud_state_empty = "electrothermal_empty"
+	ammo_behavior_flags = AMMO_ENERGY|AMMO_SPECIAL_PROCESS
+	shell_speed = 0.1
+	damage = 150
+	penetration = 50
+	bullet_color = COLOR_PALE_GREEN_GRAY
+
+/datum/ammo/energy/bfg/ammo_process(obj/projectile/proj, damage)
+	if(proj.distance_travelled <= 2)
+		return
+	// range expands as it flies to avoid hitting the shooter and tank riders
+	var/bfg_range = 3
+	if(proj.distance_travelled <= 4)
+		bfg_range = (proj.distance_travelled - 2)
+	bfg_beam(proj, bfg_range, damage, penetration)
```

Comment:
This kinda makes it nearly impossible to actually get a good shot. CLose xenos cant get hit, and further away xenos can just walk offscreen because the ball moves so slowly.